### PR TITLE
Get motions by party

### PR DESCRIPTION
**Pull request by yvonnele**
- Implemented getMotionsByParty which frontend will show

### CODE REVIEW

File: wdip-be/api/getMotionsByParty.js [hunk 1]
```diff
@@ -1,14 +1,63 @@
...
+    return dbClient.msearch({
+      body: toAddDocs
+    });
+  } catch (error) {
+    logger.error("Error fetching documents in getMotions: ", error);
+  }
+}
+
+module.exports = async function getMotionsByParty(fromDateStrOverride = new Date(2000, 0, 1), toDateStrOverride = new Date(2018, 0, 1), parties = DEFAULT_PARTIES) {
+  const posResponse = getMotions(fromDateStrOverride, toDateStrOverride, "bifall", parties);
```
Comment (henriksvensson) [inline wdip-be/api/getMotionsByParty.js]: Maybe I'm missing something, but I was expecting us to be able to run a query against ES and get the aggregated data back in one call. If I understand the code correctly, we make searches for each party and for each verdict (bifall/avslag) before we aggregate it ourselves. But maybe that's necessary?

Comment (yvonnele) [reply wdip-be/api/getMotionsByParty.js]: It is divided into two calls right now, one call for aggregated data (ES does this) on approved motions and the second one for declined motions. It's possible to group the queries from the two queries into one single call, but i thought this way would be easier if we would like to query for a new status in the future. But maybe it's not necessary if avslag and bifall are the only ones we would like to count?

I looked into terms aggregation at first (https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html) but the value we want (document counts) are apparently only approximates. Maybe there are some other ways that I have missed!

Comment (henriksvensson) [reply wdip-be/api/getMotionsByParty.js]: Ah, ok. It's my old SQL background that keeps getting in the way =).

Comment (yvonnele) [reply wdip-be/api/getMotionsByParty.js]: There are probably more optimised ways to write this. It sparked a chain of thought, e.g. combining (https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-metrics-valuecount-aggregation.html) with some kind of filter.

**Pull request merged**